apps/agreement/test/agreement/agreement_initialize.js

```diff
@@ -70,6 +70,10 @@ contract('Agreement', ([_, EOA]) => {
         const { recipient: expectedCashier } = await arbitrator.getSubscriptionFees(ZERO_ADDRESS)
         assert.equal(setting.aragonAppFeesCashier, expectedCashier, 'aragon app fees cashier does not match')
       })
+
+      it('does not allow recovering funds', async () => {
```

Comment:
We may want to test this with a slightly more complicated example on a mock org.

You can set an EOA as the 'default vault', and then try to call `transferToVault()`.

Reply:
I couldn't set the default vault to an EOA, but I did use `transferToVault` with using a sample ERC20 :)

Reply:
Good point; the Kernel checks if the app address is a contract!